README.md

```diff
@@ -210,15 +210,23 @@ performance optimizations.
 ```
 const backfill = require("backfill/lib/api");
 
-const hashSalt = await getEnvironementHash();
+/*
```

Comment:
Actually, I think we could just replace `hashSalt` on line 223 with some string such as `"optional-salt"` and remove this comment and the getNodeVersion. I think the description and example is concise and clear, but right now it seems like the salt thing is a big deal, but it's more like a feature you can choose to use if you need it.

Reply:
I don't know what is the most confusing, providing a dummy value to a required parameter makes us wonder why this thing is here at all.

Reply:
Or maybe this discussion is a sign that this parameter should be optional.

Reply:
The salt is now optional :)